ingestion/src/metadata/orm_profiler/validations/table/table_row_count_to_be_between.py

```diff
@@ -44,18 +47,30 @@ def table_row_count_to_be_between(
         msg = "rowCount should not be None for TableRowCountToBeBetween"
         logger.error(msg)
         return TestCaseResult(
-            executionTime=execution_date.timestamp(),
+            timestamp=execution_date.timestamp(),
             testCaseStatus=TestCaseStatus.Aborted,
             result=msg,
         )
 
+    min_ = next(
```

Comment:
maybe all these `next` calls could be encapsulated in a utils functions as we are repeating it quite a bit

Reply:
@pmbrull yes, I am planing on doing that when splitting the profiler and the test if that is ok.